api/kyverno/v2beta1/exception_type.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2022 The Kubernetes authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package v2beta1
```

Comment:
I don't see we add this new type to Scheme, something like this:

https://github.com/kyverno/kyverno/blob/main/api/kyverno/v1/register.go

@Eileen-Yu - did you not push all changes?

Reply:
Hi @realshuting This works!
Actually missed adding type to the scheme.(my bad for lacking of background knowledge..)
Anyway, nice catch and thx for pointing out!